Reformat the code base with Eclipse

bsataric opened the pull request:
This PR has been created in order to change file formatting of certain files according to eclipse IDE formatting. This will help in easy comparison with task addition branch which has been formatted in the same way by Eclipse.

No actual changes to the source codes have been made appart to the formatting (indendation, bracket positioning etc).

fwyzard commented on src/confdb/converter/ConverterFactory.java:
```diff
@@ -8,7 +8,7 @@
 	private boolean untrackedFlag = false;
 	private String additionalParameter4ConfigurationWriter = null;
-
+	//DUMY COMMENT
```
I'm curious... why this ?

bsataric replied:
I added it so I could check some problems that we had trying to compare branches before this.  Just to see weather commit reflects on the branch in comparison. It should be removed.

fwyzard commented on bin/.gitignore:
```diff
@@ -0,0 +1,22 @@
+/browser/
+/build.xml
+/createKeyStore
+/deploy.sh
```
these files are part of the repository, why do you propose to ignore them ?

bsataric replied:
I haven't changed gitignore by hand from the origiinal code I branched off. I don't know why it became chaged meanwhile. I wil change it back to original as in master. My guess is that it has to do something with project being repeatedly opened in Eclipse.

fwyzard commented on bin/ChangeLog:
```diff
@@ -0,0 +1,2612 @@
+2017-06-08 Andrea Bocci <andrea.bocci@cern.ch>
```
why do you copy this from the root directory ?

bsataric replied:
Mistake I guess. i will delete it from the bin. Might have to do something with Eclipse.

fwyzard replied:
Still not deleted ?

fwyzard commented on bin/README:
```diff
@@ -0,0 +1,94 @@
+----------------
```
why do you copy this from the root directory ?

bsataric replied:
Mistake I guess. i will delete it from the bin. Might have to do something with Eclipse.

fwyzard replied:
Still not deleted ?

fwyzard commented on src/confdb/data/ConfigInfo.java:
```diff
@@ -5,238 +5,238 @@
 /**
- * ConfigInfo
- * ----------
+ * ConfigInfo ----------
```
mhm, no, this was supposed to stay on the next line

bsataric replied:
OK. Will revert manually.

fwyzard commented on src/confdb/data/Configuration.java:
```diff
@@ -7,1573 +7,1588 @@
 /**
- * Configuration
- * -------------
+ * Configuration -------------
```
same here

bsataric replied:
OK. Will revert manually.

fwyzard commented on src/confdb/data/ConfigurationModifier.java:
```diff
@@ -3,976 +3,917 @@
 /**
- * ConfigurationModifier
- * ---------------------
+ * ConfigurationModifier ---------------------
```
same here

bsataric replied:
OK. Will revert manually.

fwyzard commented on src/confdb/data/ConfigurationModifier.java:
```diff
@@ -3,976 +3,917 @@
  *
- * retrieve only selected parts of a configuraiton,
- * after applying configurable filter decisions.
+ *         retrieve only selected parts of a configuraiton, after applying
```
this in indentation does not seem correct

bsataric replied:
Will correct. I guess Ecliipse is not perfect.

fwyzard commented on src/confdb/data/DatabaseEntry.java:
```diff
@@ -3,49 +3,56 @@
 /**
- * DatabaseEntry
- * --------------
+ * DatabaseEntry --------------
```
same here

bsataric replied:
OK. Will revert manually.

fwyzard commented on src/confdb/data/DatabaseEntry.java:
```diff
@@ -3,49 +3,56 @@
- * the DB and have not changed, indicate those object which are either
- * not yet in the DB or have changed (and therefore need to have a new
- * copy saved.
+ *         Class to keep track of the status of a component in the database:
```
indentation ?

bsataric replied:
Will correct. I guess Ecliipse is not perfect.

fwyzard commented on src/confdb/gui/CreateStreamDialog.java:
```diff
@@ -11,197 +11,195 @@
 /**
- * CreateStreamDialog
- * ------------------
+ * CreateStreamDialog ------------------
```
same here

bsataric replied:
OK. Will revert manually.

fwyzard commented:
I stopped checking after the first few files... could you check the formatting of the comments ?
Some of them do not look right.

The changes in the actual code look fine.

fwyzard commented:
There is also a new binary file, ` bin/antlib/ant-contrib-1.0b3.jar`.
If it was not intended, could you delete it ?

bsataric commented:
> I stopped checking after the first few files... could you check the formatting of the comments ?
> Some of them do not look right.
> 
> The changes in the actual code look fine.

When you say comments - you mean comments in the header of the changed files or all coments anywhere in the files?

fwyzard commented:
> When you say comments - you mean comments in the header of the changed files or all coments anywhere in the files?

For example
  - https://github.com/cms-sw/hlt-confdb/pull/13/files?diff=unified&w=1#diff-a19dc91caec343a829f3aecbefd43641L8-R8
  - https://github.com/cms-sw/hlt-confdb/pull/13/files?diff=unified&w=1#diff-ebe8f24fd70b33027bdbec97406b5176L17-R17

The bulk of the code and the comments throughout the body look fine, from a quick glance.

bsataric commented:
> > When you say comments - you mean comments in the header of the changed files or all coments anywhere in the files?
> 
> For example
> 
> * https://github.com/cms-sw/hlt-confdb/pull/13/files?diff=unified&w=1#diff-a19dc91caec343a829f3aecbefd43641L8-R8
> * https://github.com/cms-sw/hlt-confdb/pull/13/files?diff=unified&w=1#diff-ebe8f24fd70b33027bdbec97406b5176L17-R17
> 
> The bulk of the code and the comments throughout the body look fine, from a quick glance.

OK. I pushed changes you proposed. Can you check if this is what you wanted? Especially regarding gitignore files because a lot of files have been added. Thanks!

fwyzard commented on bin/.gitignore:
```diff
@@ -0,0 +1,18 @@
+classes/
```
My original comment was probably wrong, sorry about that.

Why do you add this file at all ?
It should already be there in the top directory.

fwyzard commented:
`bin/antlib/ant-contrib-1.0b3.jar` and other new files are still present.
Can you remove those that are clearly there by mistake ?
About the additional xml files, which ones are actually needed/used ?

fwyzard commented:
Actually, it looks like there is a whole new `bin/browser` directory ?

Can you please remove all the new files (unless they are needed by Eclipse to build the project),  and squash away all intermediate commits to remove them from the git history ?

bsataric commented:
> Actually, it looks like there is a whole new `bin/browser` directory ?
> 
> Can you please remove all the new files (unless they are needed by Eclipse to build the project), and squash away all intermediate commits to remove them from the git history ?

OK I hope I deleted all uncessary files/folders (I tried to mimic what is already on ConfDB repo and eliminate all that is not already there). I also squashed commits into 1.

fwyzard commented on src/confdb/data/ConfigInfo.java:
```diff
@@ -5,238 +5,238 @@
-
 /**
- * ConfigInfo
+ * ConfigInfo 
```
sorry for being pedantic - could you avoid adding the extra space at the end of the line ?
(here and in the other files)

fwyzard commented:
The rest looks good, thanks for the clean up !

bsataric committed:
Eclipse formatting done
